kod/object/active/holder/room/battlearena.kod

```diff
@@ -0,0 +1,1095 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+BattleArena is Room
+
+constants:
+
+   include blakston.khd
+
+   SECTOR_DOOR_CRYPT3A = 5
+   FLAGS_TOTAL = 9
+   MINUTES_TO_WIN = 3
+
+   TEAM_SPECTATOR = 1
+   TEAM_ZANDRAMAS = 2
```

Comment:
I would define these in blakston.khd, then you can use them anywhere you need to query teams rather than using 1, 2 or 3 (which may be confusing for future contributors).